lib/components/InputBase/style.scss

```diff
@@ -9,6 +9,31 @@
 .root {
 }
 
+$active_scaling_factor: 0.7777;
+$active_padding_added: var(--global--space--2); // 8
+
+$size: 64px; // Hardcoded values are fixes for edge not supporting css variable inside transform
+$font_size: var(--global--fontSize--6); // 24
+
+@function calcPlaceholderTranslate($notched) {
+	// TODO hard-coding these values is due to edge struggling width cal and variables mixed together
+	/* stylelint-disable */
+	@if $notched == true {
+		@return translate(
+			20px,
+			// calc(var(--global--space--3) + #{$active_padding_added}),
+				-9.33324px // calc(-1 * (((#{$font_size}) * #{$active_scaling_factor}) / 2))
```

Comment:
Can this not be calculated using scss? ie `(-1 * ($font_size * $active_scaling_factor) / 2))`, and let `$font_size` be equal to a static value?